cassandra_test.go

```diff
@@ -1954,283 +1954,6 @@ func TestEmptyTimestamp(t *testing.T) {
 	}
 }
 
-// Integration test of just querying for data from the system.schema_keyspace table where the keyspace DOES exist.
-func TestGetKeyspaceMetadata(t *testing.T) {
```

Comment:
Because this PR introduces different Metadata structures for Scylla and Cassandra. These tests compiles only on cassandra, so I moved them to other suite. Similar tests are added in scylla suite.